plugins/search-index.js

```diff
@@ -1,30 +1,38 @@
 const path = require('path');
+const { getAllPosts, generateIndexSearch } = require('./util');
 
-const SearchIndexWebpackPlugin = require('./search-index-compiler');
-
-const DEFAULT_OUTPUT_DIRECTORY = './public';
-const DEFAULT_OUTPUT_NAME = 'wp-search.json';
+const WebpackPlugin = require('./plugin-compiler');
 
 module.exports = function indexSearch(nextConfig = {}) {
-  const { env, outputDirectory = DEFAULT_OUTPUT_DIRECTORY, outputName = DEFAULT_OUTPUT_NAME } = nextConfig;
+  const plugin = {
+    name: 'SearchIndex',
+    outputDirectory: './public',
```

Comment:
should this object definition be swapped with the `nextConfig` statement below?

if someone passed in `outputDirectory` to `nextConfig`, wouldn't the `plugin` value that gets passed into `WebpackPlugin` be wrong?

Reply:
My mistake. Updating now. Thanks!